topn/topn.c

```diff
@@ -917,270 +799,212 @@ int main(int argc, char **argv)
 
 void sig_handler(int signal)
 {
-   if (signal != SIGALRM)
-   {
+   if (signal != SIGALRM) {
       return;
    }
    print_stats = 1;
 }
 
-int get_array_index (uint32_t key, flow_t ** sorted_array, size_t num)
-{
+int get_array_index (uint32_t key, flow_t **sorted_array, size_t num) {
    if (sorted_array[0]->max_number > key) return -1;
 
-   /** 
+   /**
    * In case sorted_array has less than 4 elements, I process it manually.
    * It's because main algorithm (see below) doesn't work well for 1, 2 and 3 members.
    */
 
-   if (num == 1)
-   {
+   if (num == 1) {
       if (sorted_array[0]->max_number > key) return -1;
       else return 0;
    }
 
-   if (num == 2)
-   {
+   if (num == 2) {
       if (sorted_array[0]->max_number > key) return -1;
```

Comment:
Do not leave the statement of "true" branch of `if` without brackets, always use:
```
if () {
}
```
even for just one statement.